src/components/header/header.jsx

```diff
@@ -101,6 +101,15 @@ const Header = () => {
                         {/*--------------------------------------------------------------------------------*/}
                     </Nav>
                     <Nav className="ml-auto float-right" navbar>
+                    <NavItem>
+                            <a
+                                href="/#/new_listing"
+                                className="btn btn-warning mr-2"
+                                style={{ marginTop: '20px' }}
+                            >
+                                List an item
+                </a>
+                        </NavItem>

```

Comment:
Will this appear on the sidebar? In addition, at this point in the code are there still lots of headers that we are not using in the sidebar? Other than that LGTM this will be important to be able to list an item once frontend and backend are connected.

Reply:
This is a button that appears in the header (top of the page), the code for it to appear in the sidebar is located in /src/routes/routing.jsx . At this point there were additional sidebar tabs that would take you to additional pages. I believe that we have removed those already within Master as well.